### PR TITLE
Clarify identity platform CNAME behaviour

### PR DESCRIPTION
**Pull request by PaulMartinsen**
Added information to help users understand and resolve obscure O365 license activation errors.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/external-domain-name-system-records.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Every Office 365 customer needs to add two records to their external DNS. The fi
...
+|**CNAME** <br/> **(Suite)** <br/> |Used by Office 365 to direct authentication to the correct identity platform. [More information](../admin/services-in-china/purpose-of-cname.md?viewFallbackFrom=o365-worldwide) <br/> **Note:** This CNAME only applies to Office 365 operated by 21Vianet. [More information](/office365/servicedescriptions/office-365-platform-service-description/office-365-operated-by-21vianet) If present and your Office 365 is not operated by 21Vianet users on your custom domain will get a "*custom domain* isn't in our system" error and won't be able to activate their Office 365 license.  |**Alias:** msoid  <br/> **Target:** clientconfig.partner.microsoftonline-p.net.cn  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/external-domain-name-system-records.md]:
```suggestion
|**CNAME** <br/> **(Suite)** <br/> |Used by Office 365 to direct authentication to the correct identity platform. [More information](../admin/services-in-china/purpose-of-cname.md?viewFallbackFrom=o365-worldwide) <br/> **Note:** This CNAME only applies to Office 365 operated by 21Vianet. If present and your Office 365 is not operated by 21Vianet, users on your custom domain will get a "*custom domain* isn't in our system" error and won't be able to activate their Office 365 license. [More information](/office365/servicedescriptions/office-365-platform-service-description/office-365-operated-by-21vianet) |**Alias:** msoid  <br/> **Target:** clientconfig.partner.microsoftonline-p.net.cn  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PaulMartinsen, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**